libcontainer/rootfs_linux.go

```diff
@@ -450,22 +450,21 @@ func mountToRootfs(c *mountConfig, m mountEntry) error {
 			}
 		}
 
+		if stat != nil {
```

Comment:
nit: maybe change this to `else` (of the previous `if err != nil`)?

Reply:
Updated.